src/Microsoft.Azure.SignalR/EndpointRouters/DefaultEndpointRouter.cs

```diff
@@ -19,8 +20,8 @@ internal class DefaultEndpointRouter : DefaultMessageRouter, IEndpointRouter
         public ServiceEndpoint GetNegotiateEndpoint(HttpContext context, IEnumerable<ServiceEndpoint> endpoints)
```

Comment:
Shall we add UT to cover the method as now complicated logic introduced here?

Reply:
Add ut EndpointRouterTest